app/controllers/rents_controller.rb

```diff
@@ -6,15 +6,17 @@ def rent_params
     params.permit(:user_id, :book_id, :from, :to)
   end
 
-  # El codigo esta asi para que pase los tests.
-  # TODO: si entran por user_id o por book_id es la misma url. Como identificarlo?
+  def rents(params)
+    if params[:user_id].present?
+      authorize Rent.includes(:user, :book).where(user_id: params[:user_id])
```

Comment:
Since `Rent.includes(:user, :book).where(user_id: params[:user_id])` is repeated twice, extract it to a local variable